standards/docker_standards.md

```diff
@@ -0,0 +1,230 @@
+## Docker standards
+
+### General
+- all containers should be created using Docker
+- Linux containers should be used
```

Comment:
I have heard some discussion about possibly containerising some Windows services - do we _have_ to stipulate Linux here?

Reply:
I guess it depends on your hosting environment.  A machine will likely only be capable of Windows or Linux containers, I was trying to make life easier by saying we should stick to Linux and we won't have unexpected platform issues.  Linux containers are fine for .Net Core services.  However, if we are planning on containerising .Net Framework then that would only work in Windows containers.

I'll change to "Linux containers are preferred".